src/app.ts

```diff
@@ -0,0 +1,14 @@
+import { envs } from "./core/config/env";
+import { Server } from "./server";
+
+(() => {
+	main();
+})();
```

Comment:
You should have the same result just calling main (after the function declaration);
```suggestion
main();
```